prov/rxm/src/rxm_attr.c

```diff
@@ -75,10 +75,7 @@ struct fi_domain_attr rxm_domain_attr = {
 	.data_progress = FI_PROGRESS_AUTO,
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
-	/* Advertise support for FI_MR_BASIC so that ofi_check_info call
-	 * doesn't fail at RxM level. If an app requires FI_MR_BASIC, it
-	 * would be passed down to core provider. */
-	.mr_mode = FI_MR_BASIC,
+	.mr_mode = FI_MR_BASIC | FI_MR_SCALABLE,
```

Comment:
I think we should keep the comment here. Without the comment it looks as if RxM supports basic and scalable modes intrinsically when in reality it is dependent on the core provider.